src/cards/bleed.ts

```diff
@@ -36,7 +36,7 @@ const spell: Spell = {
     manaCost: 10,
     healthCost: 0,
     expenseScaling: 1,
-    probability: probabilityMap[CardRarity.RARE],
+    probability: probabilityMap[CardRarity.UNCOMMON],
```

Comment:
Bleed is a fairly potent spell, why is it made more common?  This makes it 3x more likely to appear

Reply:
I did this because bleed is fun and it feels like I never see it/get to use it.

I don't feel too strongly about this change, just thought it was nice to give more non-slash options. It is a very powerful spell for "UNCOMMON", to be fair.

Reply:
I feel like it's a pretty critical end game build spell.  I think we should leave it Rare, simply because it's so potent and it occurring less frequently will entice people to take it when they see it and use it

Reply:
That makes sense!